test-snap-build/README.md

```diff
@@ -0,0 +1,28 @@
+# snapcrafters/ci/test-snap-build
+
+This action tries to build the snap "locally" on the Github Actions runner for `amd64` only. Once
+the snap is built, it is reviewed using [review-tools](https://snapcraft.io/review-tools). Designed
+to be a quick "smoke test" that doesn't require any special credentials or secrets.
```

Comment:
Maybe explain here why we don't use `remote-build`? For example:

> Note: we don't use `remote-build` here, because that requires access to a Launchpad token. Exposing tokens in a PR build [is dangerous from a security standpoint](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/).